### PR TITLE
Theming v2 channel list and preview react

### PR DESCRIPTION
**Pull request by MartinCupela**
### 🎯 Goal

- add React specific changes to v2 styles
- add `VirtualMessageList` styles
- add `MessageNotifications` styles

### CODE REVIEW

File: src/v2/styles/ChannelPreview/ChannelPreview-layout.scss [hunk 1]
```diff
@@ -8,7 +8,7 @@
   width: 100%;
   padding: var(--str-chat__spacing-3) var(--str-chat__spacing-2);
   margin: 0;
-  text-align: start;
+  text-align: flex-start;
```
Comment (szuperaz) [inline src/v2/styles/ChannelPreview/ChannelPreview-layout.scss:11]: @MartinCupela I think this should stay `start`

File: src/v2/styles/MessageList/MessageList-layout.scss [hunk 2]
```diff
@@ -37,6 +36,23 @@
...
+  max-height: 100%;
+
+  .str-chat__message-list-scroll {
+    @include utils.message-list-spacing;
+  }
+
+  .str-chat__list__center {
+    @include utils.flex-row-center;
+    width: 100%;
+  }
```
Comment (szuperaz) [inline src/v2/styles/MessageList/MessageList-layout.scss:53]: @MartinCupela doesn't React needs the rules defined for `str-chat__ul` and so on (defined above)?

File: src/v2/styles/MessageNotifications/MessageNotifications-theme.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
+@use '../utils';
+
+.str-chat {
+  --str-chat__message-notification-font-family: var(--str-chat__font-family);
```
Comment (szuperaz) [inline src/v2/styles/MessageNotifications/MessageNotifications-theme.scss:4]: I think it's good to add the documentation as well, it's harder to do it later

File: src/v2/styles/MessageList/MessageList-layout.scss (hunk 2 above)
Comment (MartinCupela) [reply src/v2/styles/MessageList/MessageList-layout.scss:53]: yes, will add, thank you :)

File: src/v2/styles/ChannelPreview/ChannelPreview-layout.scss (hunk 1 above)
Comment (MartinCupela) [reply src/v2/styles/ChannelPreview/ChannelPreview-layout.scss]: fixed

File: src/v2/styles/MessageNotifications/MessageNotifications-theme.scss (hunk 3 above)
Comment (MartinCupela) [reply src/v2/styles/MessageNotifications/MessageNotifications-theme.scss:4]: Thank you, variables for `MesssageNotifications` are documented now

**Review by oliverlaz — APPROVED** [review]

**Pull request merged**